mechanics/physicell/tests/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+find_package(GTest REQUIRED)
```

Comment:
please copy `reactions-diffusion/biofvm/tests/.clang-tidy` to this tests directory to filter some linter checks